src/Migration/Transfer.php

```diff
@@ -40,11 +40,6 @@ class Transfer
         Resource::TYPE_INDEX,
         Resource::TYPE_COLUMN,
         Resource::TYPE_ROW,
-
-        // legacy
-        Resource::TYPE_DOCUMENT,
-        Resource::TYPE_ATTRIBUTE,
-        Resource::TYPE_COLLECTION,
```

Comment:
Having these here just duplicates what were store in the migrations `resources`, we now the have fallbacks to handle existing migrations that reference these